app/include/zmk/hid.h

```diff
@@ -108,40 +108,21 @@ static const u8_t zmk_hid_report_desc[] = {
     0x00,
     /* LOGICAL_MAXIMUM (1) */
     HID_GI_LOGICAL_MAX(1),
-    0x01,
-    /* USAGE (Scan Next Track) */
-    HID_LI_USAGE,
-    0xB5,
-    /* USAGE (Scan Previous Track) */
-    HID_LI_USAGE,
-    0xB6,
-    /* USAGE (Stop) */
-    HID_LI_USAGE,
-    0xB7,
-    /* USAGE (Eject) */
-    HID_LI_USAGE,
-    0xB8,
-    /* USAGE (Media Play/Pause) */
-    HID_LI_USAGE,
-    0xCD,
-    /* USAGE (Mute) */
-    HID_LI_USAGE,
-    0xE2,
-    /* USAGE (Volume Increment) */
-    HID_LI_USAGE,
-    0xE9,
-    /* USAGE (Volume Decrement) */
-    HID_LI_USAGE,
-    0xEA,
+    0xFF,
+    HID_LI_USAGE_MIN(1),
+    0x00,
+    /* USAGE_MAXIMUM (Keyboard Application) */
+    HID_LI_USAGE_MAX(1),
+    0xFF,
     /* INPUT (Data,Ary,Abs) */
-    /* REPORT_SIZE (1) */
+    /* REPORT_SIZE (8) */
     HID_GI_REPORT_SIZE,
-    0x01,
+    0x08,
     /* REPORT_COUNT (8) */
     HID_GI_REPORT_COUNT,
-    0x08,
+    0x06,
```

Comment:
For consumer page stuff, I implemented "6 key roll over" only, since that's all we really need for this. We could even get away with "4KRO" if we wanted, I don't expect a need for holding a lot of consumer keys at once.